.github/workflows/e2e-nvidia-a10g-x1.yml

```diff
@@ -156,7 +156,7 @@ jobs:
 
           python3.11 -m pip show nvidia-nccl-cu12
           
-          ./scripts/basic-workflow-tests.sh -emf
+          ./scripts/basic-workflow-tests.sh -mf
```

Comment:
The reason for this change was that MT-Bench-Branch uses merlinite-7b-lab as the base model. Merlinite-7b-lab cannot be served on a single A10 GPU with vLLM unless `--gpu-memory-utilization 1` is passed into vLLM. Currently the eval code doesn't have the ability to pass abitrary vLLM flags to individual models within MT-Bench to get merlinite to run.

CC @nathan-weinberg @alinaryan

Reply:
Can we just change the model to granite?